buildozer/targets/ios.py

```diff
@@ -99,10 +102,21 @@ def install_platform(self):
                                                           branch='1.7.0',
```

Comment:
I decided to not address this DRY now, but it's on my radars for next pull requests